pyuvsim/simsetup.py

```diff
@@ -4,29 +4,29 @@
 
 from __future__ import absolute_import, division, print_function
 
-import numpy as np
-from numpy.lib import recfunctions
-import yaml
+import ast
+import copy
 import os
 import shutil
 import warnings
-import six
-import ast
-import copy
-from six.moves import map, range, zip
+
 import astropy.units as units
-from astropy.time import Time
+import numpy as np
+import pyuvdata.utils as uvutils
+import six
+import yaml
+from astropy.coordinates import Angle, SkyCoord, EarthLocation
 from astropy.io.votable import parse
-from astropy.coordinates import Angle, SkyCoord, EarthLocation, AltAz
-
+from astropy.time import Time
+from numpy.lib import recfunctions
 from pyuvdata import UVBeam, UVData
-import pyuvdata.utils as uvutils
+from six.moves import map, range
```

Comment:
I'd like to see imports from the same library next to each other. I find that *much* more readable.

Reply:
absolutely agree. check it in the new commit when I push it.